src/machine/machinectl.c

```diff
@@ -2457,6 +2457,7 @@ static int help(int argc, char *argv[], void *userdata) {
                "  status NAME...              Show VM/container details\n"
                "  show [NAME...]              Show properties of one or more VMs/containers\n"
                "  start NAME...               Start container as a service\n"
+               "  stop  NAME...               Stop container (equal to poweroff)\n"
```

Comment:
I think this shouldn't be mentioned in --help at all. It's sufficientto mention this in --help